types/index.d.ts

```diff
@@ -827,6 +851,7 @@ export class KafkaJSUnsupportedMagicByteInMessageSet extends KafkaJSError {
 }
 
 export class KafkaJSDeleteGroupsError extends KafkaJSError {
+  readonly groups: any[];
```

Comment:
This should be:

```suggestion
  readonly groups: DeleteGroupsResult[];
```

Except the `DeleteGroupsResult` is actually not correctly typed. It should be:

```
export type DeleteGroupsResult = {
  groupId: string
  errorCode?: number
  error?: KafkaJSProtocolError
}
```

Reply:
will fix up `DeleteGroupsResult` type as well.